src/alfasim_score/converter/alfacase/convert_alfacase.py

```diff
@@ -0,0 +1,49 @@
+from alfasim_sdk import AnnulusDescription
+from alfasim_sdk import FormationDescription
+from alfasim_sdk import ProfileDescription
+from alfasim_sdk import WellDescription
+from alfasim_sdk import XAndYDescription
+from barril.units import Scalar
+
+from alfasim_score.constants import ANNULUS_TOP_NODE_NAME
+from alfasim_score.constants import WELLBORE_BOTTOM_NODE
+from alfasim_score.constants import WELLBORE_TOP_NODE
+from alfasim_score.converter.alfacase.score_input_reader import ScoreInputReader
+from alfasim_score.units import LENGTH_UNIT
+
+
+class AlfacaseConverter:
```

Comment:
```suggestion
class ScoreAlfacaseConverter:
```
How about naming this class `ScoreAlfacaseConverter`? 
Just to avoid problems in the future if we need to create other converters.

Reply:
It looks better. 👍